docs/modules/ROOT/pages/documentation/payara-server/logging/logging.adoc

```diff
@@ -32,3 +32,17 @@ image:logging/logging_setup.png[Logger Settings]
 
 xref:/documentation/payara-server/logging/daily-log-rotation.adoc[Daily log rotation] +
 Documentation about daily log rotation in Payara and how to configure it.
+
+[[env-var-replacement]]
+== ENV variable replacement
```

Comment:
I would write "environment variables" everywhere instead of "ENV variables" to make it clearer.